pkg/controller/manager/node/controller.go

```diff
@@ -0,0 +1,60 @@
+package node
+
+import (
+	"context"
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+
+	networkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/apis/network.harvester.cattle.io/v1alpha1"
+	"github.com/rancher/harvester-network-controller/pkg/config"
+	"github.com/rancher/harvester-network-controller/pkg/controller/common"
+	ctlnetworkv1alpha1 "github.com/rancher/harvester-network-controller/pkg/generated/controllers/network.harvester.cattle.io/v1alpha1"
+)
+
+const (
+	controllerName = "node-controller"
```

Comment:
The name is set to finalizers of the object, so it's better to use a more concrete name to avoid conflicts with other controllers